### DISCUSSION TITLE
How to switch between various RCP API versions?

### DISCUSSION THREAD

**Discussion by tandelprashant** [openthread/openthread#8162] [Q&A] [answered]
I am not able to find the release tags or branches which can help me switch to desired RCP API version.
The latest RCP API version seems to be 6. I want to switch it to version 4 but finding it very difficult to do so.

The last tag that I could find was created on "26 Apr 2021". Not able to find any other release tags after this date.

Thanks in Advance!

**Comment by jwhui** [comment, marked as answer]
You can use `git blame` to identify the commits that changed `SPINEL_RCP_API_VERSION`.

https://github.com/openthread/openthread/blame/main/src/lib/spinel/spinel.h#L380

If possible, I would suggest updating to the latest OpenThread implementation since it includes the latest Thread 1.3 features and bug fixes.

**Comment by tandelprashant** [comment]
Thanks for the reply. The reason for not using latest version is because the RPi BR that I have is expecting older version.
I was thinking it would be great to have an option switch any version by selecting tags.
But I think for now, I can use git blame and identify the version.

Thanks again!
